### DISCUSSION TITLE
git submodule exited with code 1

### DISCUSSION THREAD

**Discussion by zhongguoliliang** [Cyfrin/foundry-full-course-cu#1569] [General]
After I type the command “forge install smartcontractkit/chainlink-brownie-contracts --no-commit” on visual studio code , it show the error “ git submodule exited with code 1”, how should I solve the problem, thank you!
![IMG_2613](https://github.com/Cyfrin/foundry-full-course-f23/assets/8157780/89d20493-be8e-457c-b69d-5906bce3164e)

**Comment by EngrPips** [comment]
Hello @zhongguoliliang , maybe you should try installing a specific version and let's see if it would work , use the command below 

```javascript
forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit
```
I am looking forward to hearing back from you.

**Reply by zhongguoliliang** [reply]
Ok,thank you very much!

**Reply by EngrPips** [reply]
Anytime buddy
